utils/unittest/epoch_builder.go

```diff
@@ -50,15 +51,20 @@ func (epoch EpochHeights) SetupRange() []uint64 {
 
 // CommittedRange returns the range of all heights in the committed phase.
 func (epoch EpochHeights) CommittedRange() []uint64 {
-	return []uint64{epoch.Committed}
+	var heights []uint64
+	for height := epoch.Committed; height < epoch.CommittedFinal; height++ {
```

Comment:
```suggestion
	for height := epoch.Committed; height <= epoch.CommittedFinal; height++ {
```

If `CommittedFinal` is the final height of the committed phase, this should include it